flask_ask/core.py

```diff
@@ -75,20 +67,15 @@ def __init__(self, app=None, route=None, blueprint=None, stream_cache=None, path
         self._launch_view_func = None
         self._session_ended_view_func = None
         self._on_session_started_callback = None
-        self._default_intent_view_func = None
         self._player_request_view_funcs = {}
         self._player_mappings = {}
         self._player_converts = {}
         if app is not None:
-            self.init_app(app, path)
+            self.init_app(app)
         elif blueprint is not None:
-            self.init_blueprint(blueprint, path)
-        if stream_cache is None:
-            self.stream_cache = SimpleCache()
-        else:
-            self.stream_cache = stream_cache
+            self.init_blueprint(blueprint)
 
-    def init_app(self, app, path='templates.yaml'):
```

Comment:
This removes the ability for the client to specify a different template. Can you tell me your thoughts on this change @johndias?